src/wrapper/HumeClient.ts

```diff
@@ -2,16 +2,19 @@ import { HumeClient as FernClient } from "../Client";
 import { ExpressionMeasurement } from "./expressionMeasurement/ExpressionMeasurementClient";
 
 export declare namespace HumeClient {
-    export interface Options extends FernClient.Options {
-        secretKey?: string;
-    }
+    type Options = FernClient.Options & { secretKey?: string } & (
+            | { accessToken: FernClient.Options["accessToken"] }
```

Comment:
i'm unable to double check right this moment but is this type from FernClient.Options possibly `| undefined`

should we pass it through a `NonNullable<T>` if so

Reply:
Great catch.

Before the NonNullable
<img width="1144" alt="image" src="https://github.com/user-attachments/assets/5c2350a6-cf9d-45b6-b05b-2d2066fab725">

After the NonNullable
<img width="1144" alt="image" src="https://github.com/user-attachments/assets/a832e677-ccf9-44c8-81f6-db0a262c4fce">

Reply:
Fixed in [No undefined, please](https://github.com/HumeAI/hume-typescript-sdk/pull/190/commits/3d1ffac0fb81c60e05a7fc2a9c9d8e816074f4f6)